heron/uploaders/src/java/com/twitter/heron/uploader/s3/S3Uploader.java

```diff
@@ -199,7 +199,7 @@ public URI uploadPackage() throws UploaderException {
       return resourceUrl.toURI();
     } catch (URISyntaxException e) {
       throw new UploaderException(
-          String.format("Could not convert URL %s to URI", resourceUrl, e));
+          String.format("Could not convert URL %s to URI", resourceUrl));
```

Comment:
We still want to pass the caught exception to UploaderException, we just should not have had it as part of the string format.